backend/test/StateChangeEvents/NightStateChangeEvents.test.js

```diff
@@ -1,8 +1,8 @@
 const Client = require('socket.io-client');
-const MafiaGameMock = require('./mocks/MafiaGameMock');
-const config = require('../config.json');
-const Player = require('../domain/Player');
-const roles = require('../domain/Enum/Role');
+const MafiaGameMock = require('../Mocks/MafiaGameMock');
```

Comment:
Either this "Mocks" needs to be lower case, or the folder needs to be changed in order for the tests to be run

Reply:
Oh man, git did not detect the change